landbosse/model/ErectionCost.py

```diff
@@ -9,7 +9,7 @@
 
 # constants
 km_per_m = 0.001
-hr_per_min = 1/60
+hr_per_min = 1 / 60
```

Comment:
I know this was primarily some automated reformatting, but one of these should be a float.